homeassistant/components/foscam/strings.json

```diff
@@ -11,7 +11,8 @@
           "stream": "Stream"
         },
         "data_description": {
-          "host": "The hostname or IP address of your Foscam camera."
+          "host": "The hostname or IP address of your Foscam camera.",
+          "rtsp_port": "New model cameras only support RTSP ports 88 and 554; old model cameras only support ports 88 and 65534."
```

Comment:
Can we first explain what rtsp port means? Data description is meant to explain what the field of supposed to be filled with. After that expansion i think we can put this text